music_assistant/providers/qobuz/__init__.py

```diff
@@ -389,20 +416,20 @@ async def library_remove(self, prov_item_id, media_type: MediaType):
                 result = await self._get_data("playlist/delete", playlist_id=prov_item_id)
             else:
                 result = await self._get_data("playlist/unsubscribe", playlist_id=prov_item_id)
-        return result
+        return result is not None
```

Comment:
did you check what was returned ?

Reply:
Tested with a Qobuz account - successful delete operations return {'status': 'success'}. Since _get_data raises exceptions on API errors and never returns None on success, return result is not None correctly returns True for successful operations.